src/analytics/lambdas/custom-resource/create-schemas.ts

```diff
@@ -386,11 +387,12 @@ async function createViewForReporting(props: ResourcePropertiesType, biUser: str
       table_event_parameter: odsTableNames.event_parameter,
       table_user: odsTableNames.user,
       table_item: odsTableNames.item,
+      viewVersion: REPORTING_VIEW_VERSION,
       ...SQL_TEMPLATE_PARAMETER,
     };
 
     for (const viewDef of props.reportingViewsDef) {
-      views.push(viewDef.sqlFile.replace('.sql', ''));
+      views.push(viewDef.sqlFile.replace('.sql', '') + REPORTING_VIEW_VERSION);
```

Comment:
In the next version, just a few SQLs are changed. What do we do?

Reply:
update all views version

Reply:
I'm against this design to introduce lots of unnecessary new MVs for every upgrade. I prefer to directly add the initial version to the file name to identify the new or changed MVs.